### PR TITLE
UTOPIA-255: Pull CSS in SASS folder

### PR DESCRIPTION
**Pull request by antsand**
# Description

This PR includes the following proposed change(s):

UTOPIA-255: Pull CSS in SASS folder

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [x] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [x] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

### CODE REVIEW

**Review by binggaofw — COMMENTED** [review]
Do we need .sass-cache folder under sass folder?

**Comment by akroon3r** [comment]
@antsand please follow the examples of previous PRs with respect to naming. If there isn't a ticket for this work, please create one on the JIRA board please.

File: src/frontend/src/sass/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.sass-cache/
```
[the remark this answers is not in the document]
Comment (Bottle7) [reply src/frontend/src/sass/.gitignore:1]: I prefer one, personally.

[the remark this answers is not in the document]
Comment (antsand) [reply src/frontend/src/sass/.gitignore:1]: Will remove it.

[the remark this answers is not in the document]
Comment (akroon3r) [reply src/frontend/src/sass/.gitignore]: Just talked with Brandon about this. I'd really like to maintain a project level .gitignore at root of the project.

Sounds like that's the direction we'd like to drive @binggaofw. Definitely a good call out and catch.

File: src/frontend/.gitignore [hunk 2]
```diff
@@ -22,3 +22,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+*.scssc
```
Comment (akroon3r) [inline src/frontend/.gitignore:25]: based on @binggaofw's feedback, lets collapse the gitignore in the frontend to a .gitignore at the project root directory.

thoughts?v

**Review by kushal-arora-fw — COMMENTED** [review]
Apart from changes Bing mentioned about `.gitignore`, looks good to me overall.

However, when you find time - can we remove the `^` from all dependencies in package.json - again to avoid unforeseen issues with version upgrades in the future?

You can refer `src/backend/package.json` if you want. Also, I am open to discussion to this ideology.

File: src/frontend/.gitignore (hunk 2 above)
Comment (Bottle7) [reply src/frontend/.gitignore:25]: I'm on it.

**Review by Bottle7 — APPROVED** [review]
This looks good to me now.

**Review by kushal-arora-fw — APPROVED** [review]
Looks good!

File: .gitignore [hunk 3]
```diff
@@ -19,10 +19,14 @@ src/backend/dist
...
 
+
+###############################################################################
+##                                  Web App                                  ##
+###############################################################################
 # Logs
-logs
-*.log
-npm-debug.log*
+src/frontend/logs
```
Comment (antsand) [inline .gitignore:27]: Not sure. This could be for both frontend and backend. They both are using npm.

**Pull request merged**